protocol-definitions/custom/Custom.yaml

```diff
@@ -868,5 +868,74 @@ customTypes:
           type: Data
           nullable: true
           since: 2.0
-
-
+  - name: SqlQueryId
+    since: 2.1
+    params:
+      - name: memberIdHigh
+        type: long
+        nullable: false
+        since: 2.1
+      - name: memberIdLow
+        type: long
+        nullable: false
+        since: 2.1
+      - name: localIdHigh
+        type: long
+        nullable: false
+        since: 2.1
+      - name: localIdLow
+        type: long
+        nullable: false
+        since: 2.1
+  - name: SqlError
+    since: 2.1
+    params:
+      - name: code
+        type: int
+        nullable: false
+        since: 2.1
+      - name: message
+        type: String
+        nullable: true
+        since: 2.1
+      - name: originatingMemberId
+        type: UUID
+        nullable: false
+        since: 2.1
+  - name: SqlColumnMetadata
+    since: 2.1
+    returnWithFactory: true
+    params:
+      - name: name
+        type: String
+        nullable: false
+        since: 2.1
+      - name: type
+        type: int
+        nullable: false
+        since: 2.1
+  - name: SqlRowMetadata
+    since: 2.1
+    params:
+      - name: columns
+        type: List_SqlColumnMetadata
+        nullable: false
+        since: 2.1
+  - name: SqlPageRow
```

Comment:
This one is also a single parameter custom type. Can you just use List_Data instead of `SqlPageRow` and remove this custom codec

Reply:
Perhaps you noticed this in some outdated version. The is no such type at the moment.